src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Stuart Haber
+      key: A
+      order: 0
+    - content: Scoot Storneta
+      key: B
+      order: 1
+    - content: Satoshi Nakamoto
+      key: C
+      order: 2
+    - content: Adam Back
+      key: D
+      order: 3
+- uuid: 7e6aa6ff-d657-4d21-982e-fea86321c613
+  type: SingleChoice
+  content: Bitcoin is backed by which of the following assets?
+  hint: NoHint
+  explanation: Bitcoin does not have a tangible value and hence is not backed by any valueable asset.
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Gold
+      key: A
+      order: 0
+    - content: USD
+      key: B
+      order: 1
+    - content: Silver
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: 1c490831-d92a-449f-b5a2-08b453cc357f
+  type: SingleChoice
+  content: Who is the creator of Bitcoin?
+  hint: NoHint
+  explanation: Bitcoin was created by Satoshi Nakamoto.
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Adam Back
+      key: A
+      order: 0
+    - content: Cynthia Dwork
+      key: B
+      order: 1
+    - content: Moni Naor
+      key: C
+      order: 2
+    - content: Satoshi Nakamoto
+      key: D
+      order: 3
+- uuid: 3110ead9-fde0-43c0-8007-b1ba3a6781fd
+  type: SingleChoice
+  content: Like every digital asset Bitcoin has the problem of “double spend”?
+  hint: Double Spend means copying and reselling the same digital asset many number of times.
+  explanation: Every bitcoin has a unique hash pointer attached to it hence it becomes impossible to create a copy with a hash present in the system.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1
+- uuid: 552462b9-be2a-4d45-b3fd-665c86c799bc
+  type: MultipleChoice
+  content: What are the benefits of Bitcoin over fiat currency?
+  hint: Benefits of blockchain
+  explanation: Bitcoin has an immutable ledger , allows users ownership and is portable to carry.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Immutable ledger
+      key: A
+      order: 0
+    - content: User sovereignty
+      key: B
+      order: 1
+    - content: High tangible value
+      key: C
+      order: 2
+    - content: Portability
+      key: D
+      order: 3
+- uuid: 99349427-2e10-4649-9962-6b26fac55f05
+  type: MultipleChoice
+  content: What are the benefits of Fiat currency over Bitcoin?
+  hint: Benefits of centralised system
+  explanation: Fiat currency has low transaction fees , they are backed by gold and thus possess a high tangible value.
```

Comment:
US dollar is not backed by gold. We can say they are `mostly backed by gold`

Reply:
Done